index.html

```diff
@@ -1,290 +1,357 @@
-                        <li>Gitlab</li>
-                        <li>Terminal</li>
-                    </ul>
-                </li>
-            </ul>
+      </section>
+    </section>
+    <!--About me section-->
+    <section class="about-me">
+      <div id="about-me">
+        <div id="head">
+          <h1>About me</h1>
+        </div>
+        <div id="text">
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
         </div>
+        <div id="resume">Get My Resume</div>
+      </div>
+      <div id="devider"></div>
+      <div class="skill">
+        <ul>
+          <li id="language">
+            <img src="Images/square-2.svg" alt="list logo" /> Language
+            <ul class="nested">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </li>
+          <br />
+          <li id="frameworks">
+            <img src="Images/square.svg" alt="second list logo" /> Frameworks
+            <ul class="nested">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </li>
+          <br />
+          <li id="skills">
+            <img src="Images/circle.svg" alt="last list logo" /> Skills
+            <ul class="nested">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
     </section>
     <!--Contact-form-section-->
     <section class="contact-form">
-        <form action="https://formspree.io/f/xayzobow" method="post" >
-            <div id="ch-container">
-                <h1 id="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
-            </div>
-        <fieldset>
+      <form action="https://formspree.io/f/xayzobow" method="post">
+        <div id="ch-container">
+          <h1 id="contact-header">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h1>
+        </div>
+        <div id="inputs">
+          <fieldset>
             <label for="fullname">
-                <input id="fullname" name="name" type="name" placeholder="Full name" maxlength="30" required />
+              <input
+                id="fullname"
+                name="name"
+                type="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
             </label>
```

Comment:
- [ ] Dear @anbehindY, Your form seems to lose its layout now. When I switch to the mobile version your form is not the same as the Figma template. Please fix this styling issue. 

### Your form on `375px` mobile
![image](https://user-images.githubusercontent.com/69077061/221260264-9218efc3-da20-4f03-8fdf-060a5e0d29d0.png)